README.md

```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
 angular.module( "myApp", [ "vokal.majora" ] );
 ```
 Include `Majora` into any modules where you want to use the bitmask service, and use the following functions to manipulate bitmasks.
-###toggleBits( number, bitsArray, newValue )
-@param {Number} number - the number to be mutated  
-@param {Number} bitsArray - selects the bits to mutate in number  
-@param {Bool} newValue - new value of the bits selected by bitsArray
-
-Use: given an input `number`, update the bits matched by `bitsArray` to be `newValue`  
-Example: Say number is 8 (binary 1000) and you want to turn on the 1st (1 or 0001) and 3rd (4 or 0100)
-least significant bits, then bitsArray would be [ 1, 4 ] and newValue would be true. The returned value
+
+## <a name="section-interface"></a>Interface
+
+The following methods can be called on the `Majora` service once injected into your Angular code.
+
+* [toggleBits( number, bitsArray, newValue )](#method-toggleBits)
+* [testOptimistic( number, bitsArray )](#method-testOptimistic)
+* [testPessimistic( number, bitsArray )](#method-testPessimistic)
+
+### Methods
+
+#### <a name="method-toggleBits"></a>`toggleBits( number, bitsArray, newValue )`
+
+Given an input `number`, update the bits matched by `bitsArray` to be `newValue`
+
+##### Arguments
+
+1. `number`    | *Number* | the number to be mutated
+
+2. `bitsArray` | *Number* | selects the bits to mutate in number
+
+3. `newValue`  | *Bool*   | new value of the bits selected by bitsArray
+
+##### Response
+
+Returns a *Number*, which is the decimal representation of the `number` argument with the selected bits modified
+
+##### Example
+
+Say `number` is 8 (binary 1000) and you want to turn on the 1st (1 or 0001) and 3rd (4 or 0100)
+least significant bits, then `bitsArray` would be [ 1, 4 ] and `newValue` would be true. The returned value
 is then 13 (binary 1101).
-###testOptimistic( number, bitsArray )
-@param {Number} number - the number to be tested  
-@param {Number} bitsArray - the bits to test in number  
 
-Use: given an input `number`, check if any of the bits in `bitsArray` are in that number  
-Example: Say number is 10 (binary 1010) and you want to check if it contains the 1st (1 or 0001)
-and 3rd (4 or 0100) least significant bits, then bitsArray would be [ 1, 4 ]. The returned value
+* * *
+
+#### <a name="method-testOptimistic"></a>`testOptimistic( number, bitsArray )`
+
+Given an input `number`, check if any of the bits in `bitsArray` are in that number
+
+##### Arguments
+
+1. `number`    | *Number* | the number to be tested
+
+2. `bitsArray` | *Number* | the bits to test in number
+
+##### Response
+
+Returns a *Bool*, which indicates whether all of the bits in `bitsArray` are active in `number`
```

Comment:
"Returns", and

    *Boolean* | indicates whether all of the bits in `bitsArray` are active in `number`